AgContext.yaml

```diff
@@ -0,0 +1,157 @@
+type: object
+required:
+  - id
+  - crop_type
+  - crop_stage
+  - background_type
+  - weather_description
+  - location_continent
+  - location_country
+  - location_state
+  - camera_model
+  - lens_model
+  - camera_height
+  - camera_angle
+  - camera_fov
+  - photography_description
+  - cropped_to_plant
+
+properties:
+
+  id:
+    type:
+      int
+      decription:
+        Integer identifying the AgContext.
+
+  crop_type:
+    type:
+      string
+      description: |-
+        General crop type
+        str. Required.
+        A string describing the general cropping scenario (e.g. "root vegetable").
+
+  crop_stage:
+    type:
+      string
+        enum:
```

Comment:
the indentation here is also broken.

Maybe you'd be more comfortable working in JSON than YAML, since it's clearer what lines up to what in JSON. Everyone else uses JSOn for JSON Schemas. I just find YAML a bit more readable (and less finicky about commas)

Reply:
JSON's bracket and whitespace heavy nature make it pretty easy for a novice like me to pick up. FYI this is my first time really doing much with YAML, and I've mostly been proceeding following Atom's queues re formatting.

Reply:
So convert it to json and continue there!